### PR TITLE
Improve messages

### PR DESCRIPTION
**Pull request by divyansh42**
Improve vulnerability details message
Improve label description to fit in GitHub UI
Signed-off-by: divyansh42 <diagrawa@redhat.com>

### Checklist

- [ ] This PR includes a documentation change
- [ ] This PR does not need a documentation change
---
- [ ] This PR includes test changes
- [ ] This PR's changes are already tested
---
- [ ] This change is not user-facing
- [ ] This change is a patch change
- [ ] This change is a minor change
- [ ] This change is a major (breaking) change

### CODE REVIEW

File: src/convert.ts [hunk 1]
```diff
@@ -194,18 +194,17 @@ function crdaToResult(
...
     const results: sarif.Result[] = [];
     const ruleIds: string[] = [];
     publiclyAvailableVulnerabilities.forEach((publiclyAvailableVulnerability) => {
         const ruleId = publiclyAvailableVulnerability.id;
-        let textMessage = `This file introduces a vulnerability ${publiclyAvailableVulnerability.title} with `
+        let textMessage = `This file introduces a ${publiclyAvailableVulnerability.title} vulnerability with `
             + `${publiclyAvailableVulnerability.severity} severity.\n`
-            + `Vulnerability present at ${dependencyName}\n`
-            + `Version: ${dependencyVersion}\n`;
+            + `Vulnerable ${isDirect ? "" : "transitive"} dependency is ${dependencyName} ${dependencyVersion}.\n`;
```
Comment (tetchel) [inline src/convert.ts:207]: if it is direct, it will say `Vulnerable  dependency` with two spaces

Comment (tetchel) [reply src/convert.ts:207]: I think it makes more sense to put the newlines here instead

File: src/convert.ts [hunk 2]
```diff
@@ -194,18 +194,17 @@ function crdaToResult(
...
+    manifestFile: string, startLine: number, isDirect: boolean,
     dependencyName: string, dependencyVersion: string, recommendedVersion: string,
     latestVersion: string,
 ): [ sarif.Result[], string[] ] {
     const results: sarif.Result[] = [];
     const ruleIds: string[] = [];
     publiclyAvailableVulnerabilities.forEach((publiclyAvailableVulnerability) => {
         const ruleId = publiclyAvailableVulnerability.id;
-        let textMessage = `This file introduces a vulnerability ${publiclyAvailableVulnerability.title} with `
+        let textMessage = `This file introduces a ${publiclyAvailableVulnerability.title} vulnerability with `
```
Comment (tetchel) [inline src/convert.ts:205]: Maybe instead of "this file" it should be "this line"

File: src/convert.ts (hunk 1 above)
Comment (divyansh42) [reply src/convert.ts:207]: You mean at the start of the string

Comment (divyansh42) [reply src/convert.ts:207]: like `\n${textMessage}Recommended Version: ${recommendedVersion}`?

Comment (tetchel) [reply src/convert.ts:207]: Yes so that when it's only the first section, there isn't an extra newline at the end.

Comment (tetchel) [reply src/convert.ts:207]: But it would be

`${textMessage}\nNew text goes here`

Comment (divyansh42) [reply src/convert.ts:207]: yeah

**Review by tetchel — APPROVED** [review]

**Pull request merged**